ledger_device_sdk/src/nbgl.rs

```diff
@@ -279,8 +200,36 @@ impl<'a> NbglHomeAndSettings<'a> {
         Reply: From<<T as TryFrom<ApduHeader>>::Error>,
     {
         unsafe {
-            if NVM_REF.is_some() {
```

Comment:
Also, I don't think we want to allow the settings to be displayed if there is no way to save them somewhere ? This can lead to some bug where the developer does not pass a reference to NVM but settings are still displayed but not working properly...

Reply:
See my previous comments but if you think it is better to deal systematically with **persistent** settings ok

Reply:
ok